docs/integrations/app-integrations/web-crawler.mdx

```diff
@@ -40,7 +45,7 @@ WHERE url = 'docs.mindsdb.com'
 LIMIT 1;
 ```
 
-You can also get the content of internal pages. Here is how to fetch the content from 10 internal pages:
+You can also retrieve content from internal pages. The following query fetches the content from 10 internal pages:
```

Comment:
By 'internal pages', are we referring to links within the link that we have provided? I am just wondering if that is clear enough?

Reply:
They are typically referred to as "subpages" or "internal pages."